charts/firehose/templ.yaml

```
@@ -0,0 +1,1125 @@
+              exec:
+                command: ["/usr/local/bin/eth-maintenance"]
+          # command:
+          #   - "bash"
+          #   - "-c"
+          #   - "tail -f /dev/null"
+          # command:
+          #   - "bash"
+          #   - "-c"
+          #   - "geth --datadir $FIREETH_READER_NODE_DATA_DIR snapshot prune-state && echo DONE && tail -f /dev/null"
+          ports:
+            - name: eth66-udp-p2p
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)